src/schoology-extractor/pyproject.toml

```diff
@@ -20,6 +20,7 @@ black = "^20.8b1"
 rope = "^0.17.0"
 flake8 = "^3.8.3"
 pytest-mock = "^3.3.1"
+pyfakefs = "^4.1.0"

```

Comment:
Were you happy with this mocking tool?

Reply:
`pyfakefs` has been great. Just lets you use the filesystem modules in `os` without actually touching the filesystem.